main.c

```diff
@@ -22,6 +22,8 @@ int main(int argc, char **argv)
     int connected = 0;
     char username[30];
     char password[30];
+    char date[20];
```

Comment:
why 20 ? make it the exact size of the date format if no reasons

Reply:
Size is now correct